src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Activity.cs

```diff
@@ -42,7 +43,7 @@ public async Task<IReadOnlyList<InstanceFileState>> UpdateInstanceWatermarkAsync
 
         logger.LogInformation("Beginning to update all instance watermarks");
 
-        IEnumerable<InstanceMetadata> instanceMetadata = await _indexStore.BeginUpdateInstancesAsync(arguments.PartitionKey, arguments.StudyInstanceUid, CancellationToken.None);
+        IEnumerable<InstanceMetadata> instanceMetadata = await _indexStore.BeginUpdateInstancesAsync(new Partition(arguments.PartitionKey, Partition.UnknownName), arguments.StudyInstanceUid, CancellationToken.None);
 
```

Comment:
Is the unknown Partition Name correct? Even though the underlying method doesn't use the name, in future if its used will have issues. What is the problem in just accept the key in this case?